.github/.cspell/flame_lottie_dictionary.txt

```diff
@@ -0,0 +1,2 @@
+Bodymovin
+xaha
```

Comment:
I don't think we need a separate dictionary for each bridge package, just add them in `flame_dictionary`.